exporter/exporter.go

```diff
@@ -12,6 +12,7 @@ type Exporter interface {
 
 type ExporterInstance interface {
 	Name() string
+	Config() Config
```

Comment:
This isn't very nice but didn't have any other ideas for the cache export to see the compressor. For other exports compressor should be set directly but for inline, it needs to use the same as what is used for the image.